SkyhookGame/Assets/Scripts/Institutions/CosmicPort.cs

```diff
@@ -1,22 +1,48 @@
-﻿using UnityEngine;
+﻿using System.Collections.Generic;
+using UnityEngine;
 
 public class CosmicPort : Institution
 {
     [SerializeField] private Vector2Int availableDocksRange = default;
     [SerializeField] private Vector2Int loadSpeedRange = default;
     [SerializeField] private Vector2Int unloadSpeedRange = default;
 
+    [Space(10f)]
+    [SerializeField] private List<Dock> allDocks = default;
```

Comment:
We can use an array since we know how many docks there are, so a list would be unnecessary.